prep-html-links/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home Page</h1>
+  <div>
+    <a href="file:///C:/Users/balta/Downloads/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
Here you want to use a relative path to the contact file instead of a full absolute path (`contact-us/contact-us.html`).